pipelines/test/tw/symlink-check.yaml

```diff
@@ -0,0 +1,31 @@
+name: symlink-check
+
+needs:
+  packages:
+    - symlink-check
+
+inputs:
+  paths:
+    description: |
+      The paths to check for broken symlinks.
+    required: false
+    default: "/"
```

Comment:
I might suggest just ditching this.  We're using '--packages' everywhere, and it is really very useful.
thoughts?

Reply:
I suspect you are wanting to use this so you can just point it at an image.  If so, then recursive scan of directory is fine. But can we just not expose that in the pipeline yaml?  

If you have a usecase in mind, then that is fine.

Reply:
I omitted it via https://github.com/chainguard-dev/tw/pull/108/commits/53d9cb0673a8d1a340fd1143a65e6e04cdd75b0d